single-sign-on/faqs.md

```diff
@@ -5,83 +5,215 @@ title: Single Sign-on FAQs
+
+### Q: Do you support IdP-initiated authentication (e.g., Okta tile support)?
+
+We do not support IdP-initiated authentication. Users must initiate login through Docker Desktop or Hub.
+
+## SSO enforcement
+
+### Q: Can I enable SSO in all organizations?
+
+You can enable SSO on organizations that are part of the Docker Business subscription.
+
+### Q: We currently have a Docker Team subscription. How do we enable SSO?
+
+Docker SSO is available with a Docker Business subscription. To enable SSO, you must first upgrade your subscription to a Docker Business subscription. To learn how to upgrade your existing account, see Upgrade your subscription.
+
+
+### Q: How does service accounts work with SSO?
+
+Service accounts work like any other user when SSO is turned on. If the service account is using an email for a domain with SSO turned on, it needs a PAT for CLI and API usage.
 
 ### Q: Is DNS verification required to enable SSO?
 
 Yes. You must verify a domain before using it with an SSO connection.
 
 ### Q: Does Docker SSO support authenticating through the command line?
 
-Yes. When SSO is enabled, you can access the Docker CLI through Personal Access Tokens (PATs).  Each user must create a PAT to access the CLI. To learn how to create a PAT, see [Managing access tokens](/docker-hub/access-tokens/). Before we transition to PATs, CLI can continue logging in using their personal credentials until early next year to mitigate the risk of interrupting CI/CD pipelines.
+Yes. When SSO is enabled, you can access the Docker CLI through Personal Access Tokens (PATs).  Each user must create a PAT to access the CLI. To learn how to create a PAT, see Manage access tokens. Before we transition to PATs, CLI users can continue logging in using their personal credentials until early next year to mitigate the risk of interrupting CI/CD pipelines.
 
-###  Q:  How does SSO affect our automation systems and CI/CD pipelines?
+### Q: How does SSO affect our automation systems and CI/CD pipelines?
 
-Before enforcing SSO, you must create PATs for automation systems and CI/CD pipelines and use the tokens instead of a password.
+Before enforcing SSO, you must create PATs for automation systems and CI/CD pipelines and use the tokens instead of a password. When SSO is enforced, password-based authentication no longer works on your automation systems and CI/CD pipelines.
 
-When SSO is enforced, password-based authentication no longer works on your automation systems and CI/CD pipelines.
+### Q: I have a user working on projects within Docker Desktop but authenticated with personal or no email. After they purchase DB licenses, they will implement and enforce SSO via Okta to manage their users. When this user signs on SSO, is their work on DD compromised/impacted with the migration to the new account?
 
-### Q: Do I need a specific version of Docker Desktop for SSO?
+If they already have their splunk email on their account, then it will be migrated to SSO.
```

Comment:
Can we update this answer to be generic? Not sure if we should mention Splunk here?

Reply:
updated